UnityClient/Assets/Scripts/TowerDefence/Enemy.cs

```diff
@@ -5,10 +5,14 @@
 
 public class Enemy : MonoBehaviour
 {
+    public float startSpeed = 10f;
+
+    [HideInInspector] public float speed;
```

Comment:
Please make this `private` and remove the `HideInInspectorAttribute`. No reason for the attribute to exist. If you need access to this value outside of this type, please use a property instead.

If it does become private, please follow the codebase's private field convention and define the name _inThisFormat, as per other places in the codebase.